python/tests/test_account.py

```diff
@@ -1525,6 +1525,81 @@ def test_send_receive_locations(self, acfactory, lp):
         locations3 = chat2.get_locations(contact=contact)
         assert not locations3
 
+    def test_undecipherable_group(self, acfactory, lp):
+        """Test how group messages that cannot be decrypted are
+        handled.
+
+        Group name is encrypted and plaintext subject is set to "..." in
+        this case, so we should assign the messages to existing chat
+        instead of creating a new one. Since there is no existing group
+        chat, the messages should be assigned to 1-1 chat with the sender
+        of the message.
+        """
+
+        lp.sec("creating and configuring three accounts")
+        accounts = [acfactory.get_online_configuring_account() for i in range(3)]
+        for acc in accounts:
+            acc.wait_configure_finish()
+            acc.start_io()
+
+        ac1 = accounts.pop()
+        lp.sec("ac1: setting up contacts with 2 other members")
+        contacts = []
+        for acc, name in zip(accounts, list("xy")):
+            contact = ac1.create_contact(acc.get_config("addr"), name=name)
+            contacts.append(contact)
+
+            # make sure we accept the "hi" message
+            ac1.create_chat_by_contact(contact)
+
+            # make sure the other side accepts our messages
+            c1 = acc.create_contact(ac1.get_config("addr"), "ä member")
+            chat1 = acc.create_chat_by_contact(c1)
+
+            # send a message to get the contact key via autocrypt header
+            chat1.send_text("hi")
+            msg = ac1._evtracker.wait_next_incoming_message()
+            assert msg.text == "hi"
+
+        ac2, ac3 = accounts
```

Comment:
with #1581 you would write:  `acfactory.get_chats(ac1, ac2) ; acfactory.get_chats(ac2, ac3) ; acfactory.get_chats(ac1, ac3)`.  This would not send any message out but would prepare all contacts.